### PR TITLE
2.0.0 "Steel Beagle"

### PR DESCRIPTION
**Pull request by marissaDubbelaar**
Conversion of the MHCquant pipeline from DSL1 to DSL2:

The main.nf includes different processes, where each process consists of one functionality
Different processes are found in modules/processes.
inclusion of workflows/mhcquant
Addition of modules.config
The openms_cometadapter also consists of the code which can be used as a module (commented since it does not work in the current state for this pipeline yet).
Change of the software version handling

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -8,197 +8,233 @@
...
-  mainScript = 'main.nf'
-  nextflowVersion = '>=19.10.0'
-  version = '1.6.0'
+    name = 'nf-core/mhcquant'
+    author = 'Leon Bichmann'
+    homePage = 'https://github.com/nf-core/mhcquant'
+    description = 'Identify and quantify peptides from mass spectrometry raw data'
+    mainScript = 'main.nf'
+    nextflowVersion = '>=21.04.0'
+    version = '2.0.0dev'
```
Comment (ggabernet) [inline nextflow.config:206]:
```suggestion
    version = '2.0.0'
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
 # nf-core/mhcquant: Changelog
 
+## v2.0.0 nf-core/mhcquant "Steel Beagle" - 2021/05/20
```
Comment (ggabernet) [inline CHANGELOG.md:3]:
```suggestion
## v2.0.0 nf-core/mhcquant "Steel Beagle" - 2021/09/03
```

Comment (ggabernet) [reply CHANGELOG.md:3]: Being optimistic with the release day :D

**Review by ggabernet — APPROVED** [review]
Looks good to me, only missing things are that the version needs to be bumped and release date updated :D

**Review by Leon-Bichmann — APPROVED** [review]
Looks good lets get this release out ;)!

**Pull request merged**